src/Apps/Order/Routes/Shipping2/Components/AddressModal2.tsx

```diff
@@ -90,15 +54,18 @@ export const AddressModal: React.FC<AddressModalProps> = ({
   const updateSavedAddress = useUpdateSavedAddress().submitMutation
   const updateUserDefaultAddress = useUpdateUserDefaultAddress().submitMutation
 
-  if (!modalAction) return null
+  if (!modalAction) {
+    return null
+  }
 
   const title = (modalAction && MODAL_TITLE_MAP[modalAction.type]) || ""
+
   const initialAddress =
     modalAction.type === "editUserAddress"
       ? modalAction.address
       : {
           // TODO: Instead of using ShippingContext, initialValues could be a shippingUtils function
-          country: shippingContext.parsedOrderData.shipsFrom,
+          country: shippingContext.orderData.shipsFrom,
```

Comment:
This is used enough throughout that we could simplify the naming a bit. The thinking is: that it is parsed is an implementation detail that future devs can cmd+click into to understand.

Reply:
yeah i thought of this but was trying to match naming from volt. Sounds better to me.